package.json

```
@@ -35,10 +35,12 @@
   "dependencies": {
     "apollo-link": "^1.2.3",
     "gatsby-source-graphql": "^2.0.5",
-    "graphql": "0.13.x",
     "pg": "^7.6.0",
     "postgraphile": "^4.1.0-rc.0"
   },
+  "peerDependencies": {
+    "graphql": "^0.13.x"
```

We should add 14 to the list too, I think, e.g.:

```suggestion
    "graphql": "^0.13.x | ^14.0.0"
```

---

What version range of GraphQL does Gatsby support?

---

Gatsby's package.json has "graphql": "^14.1.1" in its dependancies. Should we just use that?